src/Shared/Microsoft.DotNet.Services.Utilities/Utilities.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Microsoft.DotNet.Services.Utilities
+{
+    public static class Utilities
+    {
+        const string refsHeadsPrefix = "refs/heads/";
```

Comment:
Now that you are making a new file, I can complain about this. :-) It should follow the style guidelines.
```suggestion
        private const string RefsHeadsPrefix = "refs/heads/";
```